### PR TITLE
13 info box

### PR DESCRIPTION
**Pull request by DurtDiv3r**
Information composable. Used to display small amounts of information with a decorative icon. To be used in conjunction with BottomSheet.

**Work Done:**
InfoBox composable, to be used on Details screen. Conveys version information, date etc...
Update build.gradle dependency versions.

**References:**
No reference material

**Screenshot:**

InfoBox

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -21,4 +21,5 @@
     <string name="light_dark">Light/Dark</string>
     <string name="dynamic_theme">Dynamic Theme</string>
     <string name="open_options">Open options</string>
+    <string name="infobox_icon_content_desc">Information icon</string>
```
Comment (drawers) [inline app/src/main/res/values/strings.xml]: Nit: this doesn't seem to be used

File: app/src/main/java/com/islaharper/dawnofandroid/presentation/components/InfoBox.kt [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        modifier = Modifier
+            .semantics(mergeDescendants = true) {},
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Icon(
+            painter = icon,
+            contentDescription = null, // decorative
+            modifier = modifier
+                .padding(end = PADDING_SMALL)
+                .size(INFO_ICON_SIZE),
```
Comment (drawers) [inline app/src/main/java/com/islaharper/dawnofandroid/presentation/components/InfoBox.kt]: Nit, I wouldn't extract a constant in a separate file if it's only going to be used in one place. It creates separation and developers have to flip back and forth to understand the code.

Better to inline it (Ctrl Alt + N) or to put it as `private` in the same file.

**Review by drawers — APPROVED** [review]
Looks good, I left two nit pick comments. Trusting you to look at these before you merge

File: app/src/main/java/com/islaharper/dawnofandroid/presentation/components/InfoBox.kt (hunk 2 above)
Comment (DurtDiv3r) [reply app/src/main/java/com/islaharper/dawnofandroid/presentation/components/InfoBox.kt]: Refactored and committed change.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (DurtDiv3r) [reply app/src/main/res/values/strings.xml]: Refactored and committed change.

**Pull request merged**